@navikt/ds-css/baseline/normalize.css

```diff
@@ -0,0 +1,351 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
Er det ikke lurt å bruke den fra npm, så det er lettere å oppdatere? Når jeg ser inline kode som dette blir jeg usikker på om det er gjort custom endringer som jeg burde være klar over.

Reply:
`cleancss` som minimerer css-en takler ikke imports `~normalize.css` syntax for imports fra node-modules så er da enklere å ha normalize.css lokalt. Kan undersøke nærmere om eks postcss har noe som kanskje gjør dette bedre